spec/terminology.md

```diff
@@ -3,17 +3,74 @@
+[[def: authentic chained data container (ACDC), ACDC]]
+~ a variant of [the Verifiable Credential (VC) specification](https://www.w3.org/TR/vc-data-model/). Some of the major distinguishing features of ACDCs include normative support for chaining, use of composable JSON Schema, multiple serialization formats, support for Ricardian contracts, support for chain-link confidentiality compact formats for resource constrained applications, simple partial disclosure mechanisms and simple selective disclosure mechanisms. ACDCs inherit the security model derived from [[ref: KERI]].
+
+[[def: autonomic identifier (AID)]]
+~ A self-managing identifier that is self-certifying (i.e., self-authenticating) and cryptographically bound to a [[ref: key event log]] ([[ref: KEL]]). An AID is either non-transferable or transferable. A non-transferable AID does not support key rotation while a transferable AID support key rotation using a key [[ref: pre-rotation]] mechanism that enables the AID to persist in spite of the evolution of its key state.
+
+[[def: compact Event Streaming Representation (CESR), CESR]]
+~ A dual text-binary encoding format that has the unique property of text-binary concatenation composability. The composability property enables the round trip conversion en-masse of concatenated primitives between the text domain and binary domain while maintaining the separability of individual primitives.
+
+[[def: controller]]
+~ A controlling entity that can cryptographically prove the control authority (signing and rotation) over an [[ref: AID]], as well as make changes on the associated [[ref: KEL]]. A controller may consist of multiple controlling entities in a multi-signature scheme.
 
 [[def: DID document]]
 ~ As defined by [DID Core](https://www.w3.org/TR/did-core/#dfn-did-documents). See section [DID Documents](#did-documents).
 
+[[def: direct replay mode]]
+~ an operational mode of the [[ref: KERI]] protocol where a controller and a verifier of an [[ref: AID]] exchange the [[ref: KEL]] of the AID directly. Upon reception of the KEL, the verifier sends a key event receipt, which includes the verifier's signature, to the controller as an acknowledgment. The signed key event receipt attests that the verifier received and verified the key event message. 
+
+[[def: inception event]]
+~ A key establishment event that provides the incepting information needed to derive an [[ref: AID]] and establish its initial [[ref: key state]].
+
+[[def: indirect replay mode]]
+~ An operational mode of the [[ref: KERI]] protocol where the [[ref: KEL]] of an [[ref: AID]] is promulgated to a verifier using witnesses that are disignated by the AID's controller. 
+
+[[def: interaction event]]
+~ A key non-establishment event that anchors external data to the [[ref: key state]] of an [[ref: AID]] as established by the most recent prior establishment event.
```

Comment:
Given that this spec. is about `did:webs` and not about AIDs, I wondering if the `ref` here should be to the DID vs. to the AID.  I recognize that AIDs are one of the basis's for the DID, and are important to define and use, but I think in this spec, the “non-key events” are more relevant to use the use of the DID than the use of the AID.

It might be worth having an appendix that covers the AID - DID relationship, but in the terminolgy of this spec, I think we should be focused on the DID.

Reply:
Agree, I think we could be less techie, and move the rest into an addendum. Have you seen the other approach?: Brief description, more understandable for non-KERI people, and link to the KERI glossary for those who want more.

The commit is in [PR72](https://github.com/trustoverip/tswg-did-method-webs-specification/pull/72)

https://github.com/trustoverip/tswg-did-method-webs-specification/pull/72/commits/760a0144b5e7f112482d352a450bde4198c04efd